portfolio/src/main/webapp/css/main.css

```
@@ -163,4 +163,88 @@ html,h1,h2,h3,h4,h5,h6,a{
   height: 50%;
   margin: auto;
   width: 50%;
-}
\ No newline at end of file
+}
+
+/* comments style */
+.comments{
```

Style supernit: we usually put a space before the curly brace for readability. (Same throughout the file)

```suggestion
.comments {
```